### PR TITLE
END-486: new cmds for org/user balance

### PR DESCRIPTION
**Pull request by parikls**

### CODE REVIEW

**Review by asvetlov — APPROVED** [review]
Looks great, thank you!

Please add a file `./CHANGELOG.D/3137.feature` with new feature description. A sentence or two in a free form should be enough, Markdown formatting is supported if needed.

File: CHANGELOG.D/3137.feature [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Balance is no longer stored on a cluster level, and was moved to an organization level, e.g.,
+to an org itself, and to an org users, instead of a cluster / cluster users.
+
+New commands:
+  - `set_org_defaults` - allows to set an organization default values, such as a default user credits
```
Comment (asvetlov) [inline CHANGELOG.D/3137.feature:5]:
```suggestion
  - `apolo admin set-org-defaults` - allows to set an organization default values, such as a default user credits
```
`click` replaces underscores with dashes.
The full command path is more informative from end user point of view maybe, but the final decision is up to you.

The same is true for other mentioned commands.

File: CHANGELOG.D/3137.feature [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+Existing commands changes:
+  - `add_cluster_user` cmd is no longer accepting a `credits` argument.
+  - `set_user_credits` cmd is now expecting an org name instead of a cluster name.
+  - `add_user_credits` cmd is now expecting an org name instead of a cluster name.
+  - `set_org_cluster_credits` is now deprecated in a favor of a `set_org_credits`.
```
Comment (asvetlov) [inline CHANGELOG.D/3137.feature:11]: `set_org_cluster_credits` is not a thing. Do you mean `set_org_cluster_quota`?
I don't think that we need keeping deprecated admin commands, we can break backward compatibility in this command group without any problem.

Let's just drop `set_org_cluster_quota` if we don't need it. If you insist on keeping the command please add `deprecated=True` argument to `@command()` decorator.

File: CHANGELOG.D/3137.feature [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  - `add_org_cluster_credits` is not deprecated in a favor of an `add_org_credits`.
```
Comment (asvetlov) [inline CHANGELOG.D/3137.feature:12]: Sorry, I cannot find `add_org_cluster_credits` in `admin.py`

**Review by asvetlov — COMMENTED** [review]
I have a few minor notes.

Also, it would be nice of changelog record will reflect the formatters change also.

File: CHANGELOG.D/3137.feature (hunk 3 above)
Comment (parikls) [reply CHANGELOG.D/3137.feature]: @asvetlov it was removed, I've updated the changelog text.

File: CHANGELOG.D/3137.feature (hunk 2 above)
Comment (parikls) [reply CHANGELOG.D/3137.feature]: @asvetlov it was removed, I've updated the changelog text.

File: CHANGELOG.D/3137.feature (hunk 1 above)
Comment (parikls) [reply CHANGELOG.D/3137.feature]: @asvetlov changed to a full commands everywhere and used a dashes. Thanks!

**Review by asvetlov — APPROVED** [review]
Great, thanks!

**Pull request merged**